Add Mapper's Choice Awards 2024 news post (#13006)
ppy/osu-wiki: news/2025/2025-03-26-mappers-choice-awards-2024.md

- **Nomination:** Eligible users can submit any beatmaps, storyboards, or users from 2024 they believe deserve recognition. Nominated beatmaps will be eligible to enter the next stage.
- **Voting:** Eligible users vote on the nominated submissions.
- **Results:** Winners are determined by voting, with a livestream revealing the podium finishers in each category.

If you're eligible, head over to the [**Mapper's Choice Awards 2024** website](https://mca.corsace.io/2024/) to get started!

### Schedule

| Event | Date (UTC) |
| --: | :-- |
| **Nomination phase** | 2025-03-26 – 2025-03-31 (00:00) |
| **Voting phase** | 2025-04-03 – 2025-04-08 (00:00) |
| **Results stream** | 2025-04-14 |
| **Final results** | 2025-04-15 |

 View the results stream on the [Corsace Twitch channel](https://www.twitch.tv/corsace) and the final results on the [Corsace website](https://mca.corsace.io/2024/)!

### Rewards

MCA 2024 features exciting rewards for the winners of each category:

- **osu!supporter**
  - Grand Award winners: **4 months**
- **Badges**
  - Special badges for **Grand Award** category winners.
  - Generic badges for **regular category** winners.

> ### A Year in Mapping update

Reviewer: likely should be level 2 heading considering mca and ayim are presented side-by-side

```suggestion
## A Year in Mapping update
```

As we look back on 2024, the countless memories left behind in the community remain fresh in our minds. Now, an even deeper dive into the data and stats behind 2024's mapping trends awaits you in **A Year In Mapping (AYIM) 2024**!

> As always, **A Year in Mapping** serves as the mapping community's annual report, presenting [the most comprehensive and exciting stats of 2024](https://lookerstudio.google.com/u/0/reporting/aa599b20-f74b-4699-9e69-504f542de5aa/page/hj1uB?s=vvKt9IOmetM). Want to know how many Ranked maps and mappers each mode had? Or perhaps which songs and artists made the biggest impact on the community this year? It's all inside AYIM — take a look for yourself!

Reviewer 2: ```diff
- As we look back on 2024, the countless memories left behind in the community remain fresh in our minds. Now, an even deeper dive into the data and stats behind 2024's mapping trends awaits you in **A Year In Mapping (AYIM) 2024**!

- As always, **A Year in Mapping** serves as the mapping community's annual report, presenting [the most comprehensive and exciting stats of 2024](https://lookerstudio.google.com/u/0/reporting/aa599b20-f74b-4699-9e69-504f542de5aa/page/hj1uB?s=vvKt9IOmetM). Want to know how many Ranked maps and mappers each mode had? Or perhaps which songs and artists made the biggest impact on the community this year? It's all inside AYIM — take a look for yourself!
+ As we look back on 2024, the countless memories left behind in the community remain fresh in our minds. Now, an even deeper dive into the data and stats behind 2024's mapping trends awaits you in **A Year In Mapping (AYIM) 2024**! 

+ [YamYA](https://osu.ppy.sh/users/4470854) has been hosting their own iteration of AYIM for the past couple of years. And as such, we've decided to promote their work due to our previous hosts being unavailable to.

+ For those who don't know, **A Year in Mapping** serves as the mapping community's annual report, presenting the most comprehensive and exciting stats of 2024 Want to know how many Ranked maps and mappers each mode had? Or perhaps which songs and artists made the biggest impact on the community this year? It's all inside AYIM — [click here to view the most comprehensive and exciting stats of 2024!](https://lookerstudio.google.com/u/0/reporting/aa599b20-f74b-4699-9e69-504f542de5aa/page/hj1uB?s=vvKt9IOmetM)

```

@maxuskazuma fix this and should be good to go

- Credited the creator of AYIM
- Made the website link more visible

Author: changes applied with some adjustment

> As we look back on 2024, the countless memories left behind in the community remain fresh in our minds. Now, an even deeper dive into the data and stats behind 2024's mapping trends awaits you in **A Year In Mapping (AYIM) 2024**! Previous MCA AYIM has decided to not continue their service, and after some discussion, we decided to use external source to help us resume AYIM! [YamYA](https://osu.ppy.sh/users/4470854) has been hosting their own iteration of AYIM for the past couple of years. And as such, we've decided to promote their work and all credit goes to them.
> 
> For those who don't know, **A Year in Mapping** serves as the mapping community's annual report, presenting the most comprehensive and exciting stats of 2024 Want to know how many Ranked maps and mappers each mode had? Or perhaps which songs and artists made the biggest impact on the community this year? It's all inside AYIM — take a look for yourself! [click here to view the most comprehensive and exciting stats of 2024!](https://lookerstudio.google.com/u/0/reporting/aa599b20-f74b-4699-9e69-504f542de5aa/page/hj1uB?s=vvKt9IOmetM) 

Reviewer: grammar and presentation is a bit weird

```suggestion
As we look back on 2024, the countless memories left behind in the community remain fresh in our minds. Now, an even deeper dive into the data and stats behind 2024's mapping trends awaits you in **A Year In Mapping (AYIM) 2024**! For those who don't know, **A Year in Mapping** serves as the mapping community's annual report, presenting the most comprehensive and exciting stats of the past year. Want to know how many Ranked maps and mappers each mode had, or perhaps which songs and artists made the biggest impact on the community this year? It's all inside AYIM — [**click here to view the most comprehensive and exciting stats of 2024!**](https://lookerstudio.google.com/u/0/reporting/aa599b20-f74b-4699-9e69-504f542de5aa/page/hj1uB?s=vvKt9IOmetM)

The previous MCA AYIM organisers have decided to not continue their service, and after some discussion, we decided to use an external source to help us resume AYIM! [YamYA](https://osu.ppy.sh/users/4470854) has been hosting their own iteration of AYIM for the past couple of years, and as such, we've decided to promote their work with all credit going to them.
```

---
layout: post
title: Mapper's Choice Awards 2024
date: 2025-03-26 01:30:00 +0000
---

It's time to step up, vote, and crown last year's best ranked beatmaps in the **Mapper's Choice Awards (MCA)!** *This is your chance to vote for the community's stars of 2024—don’t miss out!*

> ![](link for banner)

Reviewer 3: ```suggestion
![](wiki/shared/news/Mapper_Choice_Awards_2024.jpg)
```

Reviewer 4: Image name must be changed as explained in [here](https://github.com/ppy/osu-wiki/pull/13006#discussion_r2011662884). Moreover the path needs to be absolute using `/wiki/` as root.

```suggestion
![](/wiki/shared/news/2025-03-26-mappers-choice-awards-2024/banner.jpg)
```

Folder needs to be created and the assets added as linked above.

> ---

Reviewer 5: ```suggestion
---
```

layout: post
title: Mapper's Choice Awards 2024
date: 2025-03-26 01:30:00 +0000
---

> ### Get Ready to Cast Your Vote in the Mapper’s Choice Awards 2024!  

Reviewer 5: ```suggestion
### Get Ready to Cast Your Vote in the Mapper’s Choice Awards 2024!
```

> It’s time to step up, vote, and crown last year’s best ranked beatmaps in the **Mapper’s Choice Awards (MCA)!** *This is your chance to decide who takes the top spot—don’t miss out!*

Reviewer 4: ```suggestion
It's time to step up, vote, and crown last year's best ranked beatmaps in the **Mapper's Choice Awards (MCA)!** *This is your chance to decide who takes the top spot — don’t miss out!*
```

Reviewer 6: ```suggestion
It's time to step up, vote, and crown last year's best ranked beatmaps in the **Mapper's Choice Awards (MCA)!** *This is your chance to vote for the community's stars of 2024 — don’t miss out!*
```

> It’s time to step up, vote, and crown last year’s best ranked beatmaps in the **Mapper’s Choice Awards (MCA)!** *This is your chance to decide who takes the top spot—don’t miss out!*  

Reviewer 5: ```suggestion
It’s time to step up, vote, and crown last year’s best ranked beatmaps in the **Mapper’s Choice Awards (MCA)!** *This is your chance to decide who takes the top spot—don’t miss out!*
```

> It’s time to step up, vote, and crown last year’s best ranked beatmaps in the **Mapper’s Choice Awards (MCA)!** *This is your chance to decide who takes the top spot—don’t miss out!*  

Reviewer 6: ```suggestion
It’s time to step up, vote, and crown last year’s best ranked beatmaps in the **Mapper’s Choice Awards (MCA)!** *This is your chance to decide who takes the top spot—don’t miss out!*  
It’s time to step up, vote, and crown last year’s best ranked beatmaps in the **Mapper’s Choice Awards (MCA)!** *This is your chance to vote for the community's star of 2024—don’t miss out!*
```
I think it is too superficial for MCA to just compete for the first place. I think it would be more in line with the theme of MCA if it were to select the recognized annual star of the community.

Author: okay

> ![](link for banner)  

Reviewer 5: ```suggestion
![](link for banner)
```

> For those unfamiliar with the event, the **Mapper’s Choice Awards** (*MCA*) is a **community-hosted competition** where mappers nominate and vote for the best maps and mappers across various categories, celebrating mapping excellence in all its forms.

Reviewer 4: ```suggestion
For those unfamiliar with the event, the **Mapper's Choice Awards** (*MCA*) is a **community-hosted competition** where mappers nominate and vote for the best maps and mappers across various categories, celebrating mapping excellence in all its forms.
```

> For those unfamiliar with the event, the **Mapper’s Choice Awards** (*MCA*) is a **community-hosted competition** where mappers nominate and vote for the best maps and mappers across various categories, celebrating mapping excellence in all its forms.

Author: leave as it is

> [Just like last year](https://osu.ppy.sh/home/news/2024-02-01-mappers-choice-awards-2023), **winners in each category will receive special awards!** To learn more, keep reading below and check out the [Corsace website](https://corsace.io/), where you can find past MCA results and more details about the event.  

Reviewer 5: ```suggestion
[Just like last year](https://osu.ppy.sh/home/news/2024-02-01-mappers-choice-awards-2023), **winners in each category will receive special awards!** To learn more, keep reading below and check out the [Corsace website](https://corsace.io/), where you can find past MCA results and more details about the event.
```

> [Just like last year](https://osu.ppy.sh/home/news/2024-02-01-mappers-choice-awards-2023), **winners in each category will receive special awards!** To learn more, keep reading below and check out the [Corsace website](https://corsace.io/), where you can find past MCA results and more details about the event.  

Reviewer 6: ```suggestion
[Just like last year](https://osu.ppy.sh/home/news/2024-02-01-mappers-choice-awards-2023), **Participants who win the MCA event will receive a generous reward!** To learn more, keep reading below and check out the [Corsace website](https://corsace.io/), where you can find past MCA results and more details about the event.  
```

Reviewer 4: `special awards` makes more sense in this context. 
- Generous implies plentitude. The awards are moreso unique. Special.
- Rewards are given as a form of encouragement. Awards are given as a sign of recognition. In this context, the latter is more proper to use.

Personally I'd leave it as is for the reasons listed above, plus the reword of `winners in each category` feels wordier than it should. `Winners` == the "participants who win".

Author: leave as it is

---

> ## Mapper’s Choice Awards 2024  

Reviewer 5: ```suggestion
## Mapper’s Choice Awards 2024
```

> If you **ranked a beatmap, guest difficulty, or storyboard in 2024**, you’re eligible to help the community recognise the most outstanding beatmaps of the year!

Reviewer 4: ```suggestion
If you **ranked a beatmap, guest difficulty, or storyboard in 2024**, you're eligible to help the community recognise the most outstanding beatmaps of the year!
```

> 

Reviewer 6: I think this whole section should be moved up to before ### **Key Changes for 2024**, because it is obviously more important, and these changes are not as high a priority, so I think the two parts should be swapped.

> If you **ranked a beatmap, guest difficulty, or storyboard in 2024**, you’re eligible to help the community recognise the most outstanding beatmaps of the year!  

Reviewer 5: ```suggestion
If you **ranked a beatmap, guest difficulty, or storyboard in 2024**, you’re eligible to help the community recognise the most outstanding beatmaps of the year!
```

> If you **ranked a beatmap, guest difficulty, or storyboard in 2024**, you’re eligible to help the community recognise the most outstanding beatmaps of the year!  

Reviewer 6: ```suggestion
If you **ranked a beatmap, guest difficulty, or storyboard in 2024**, you’re eligible to help the community recognise the most outstanding beatmaps and users of the year!  
```
Obviously we have always had the users award, so I think users can't be ignored.

Author: good

> ### **Key Changes for 2024**  

Reviewer 5: ```suggestion
### **Key Changes for 2024**
```

> - The number of voting categories has been **reduced**.  

Reviewer 5: ```suggestion
- The number of voting categories has been **reduced**.
```

> - The number of voting categories has been **reduced**.  

Reviewer 6: ```suggestion
- The number of voting categories has been **simplified**.  
```
Although the word simplified does not seem so intuitive, the word reduced has a slightly more derogatory connotation and is not suitable for this kind of introduction and promotion. I think simplified is more appropriate in this situation.

Reviewer 4: Original wording is fine. If anything, it is more correct. Reduced isn't derogatory (in this context). It properly states that the changes include a reduction of the number of categories. Simplified may imply that the categories either got renamed, dumbed down, or otherwise modified but not necessarily cut down in numbers.

Would leave as is.

Author: leave as it is

> - osu!supporter rewards will now be granted **only** for the **Grand Award** categories.  

Reviewer 5: ```suggestion
- osu!supporter rewards will now be granted **only** for the **Grand Award** categories.
```

> As always, MCA consists of three stages: **nomination, voting, and results.** Check the schedule below for key dates and deadlines!  

Reviewer 5: ```suggestion
As always, MCA consists of three stages: **nomination, voting, and results.** Check the schedule below for key dates and deadlines!
```

> As always, MCA consists of three stages: **nomination, voting, and results.** Check the schedule below for key dates and deadlines!  

Reviewer 6: I think this whole section should be moved up to before ### **Key Changes for 2024**, because it is obviously more important, and these changes are not as high a priority, so I think the two parts should be swapped.

Reviewer 4: Makes more sense to have changes first. Mainly to inform users already acquainted with the project what were the main differences when compared to last year's edition. That feels more important information and, as such, should remain further to the top.

Author: leave as it is

> - **Nomination:** *Eligible users can submit any beatmaps, storyboards, or users from 2024 they believe deserve recognition.*

Reviewer 6: ```suggestion
- **Nomination:** *Eligible users can submit any beatmaps, storyboards, or users from 2024 they believe deserve recognition. Nominated beatmaps will be eligible to enter the next stage.*
```

> - **Nomination:** *Eligible users can submit any beatmaps, storyboards, or users from 2024 they believe deserve recognition.*  

Reviewer 5: ```suggestion
- **Nomination:** *Eligible users can submit any beatmaps, storyboards, or users from 2024 they believe deserve recognition.*
```

> - **Nomination:** *Eligible users can submit any beatmaps, storyboards, or users from 2024 they believe deserve recognition.*  

Reviewer 6: ```suggestion
- **Nomination:** *Eligible users can submit any beatmaps, storyboards, or users from 2024 they believe deserve recognition. And the nominated maps will enter the next stage—**Vote**. *  
```

Reviewer 4: Same as https://github.com/ppy/osu-wiki/pull/13006#discussion_r2008195892 . Think something could be added here though (or better said, there's more merit to it). But the current suggestion rolls quite weirdly off the tongue.

```md
- **Nomination:** *Eligible users can submit any beatmaps, storyboards, or users from 2024 they believe deserve recognition. Nominated beatmaps will be eligible to enter the next stage.*
```

Author: will add marti's suggestion here

> - **Voting:** *Eligible users vote on the nominated submissions.*

Reviewer 6: ```suggestion
- **Voting:** *Eligible users vote on the nominated submissions. And winners are determined by voting.*  
```
The winner should be decided in the voting stage rather than the result stage. The logic here is obviously wrong.

Author: Well, the results are announced during results stage, not voting. So it makes more sense to mentioned how the winners determined in results column, as we never actually discussed anything about winners during the voting stage.

> - **Voting:** *Eligible users vote on the nominated submissions.*  

Reviewer 5: ```suggestion
- **Voting:** *Eligible users vote on the nominated submissions.*
```

> - **Voting:** *Eligible users vote on the nominated submissions.*  

Reviewer 6: ```suggestion
- **Voting:** *Eligible users vote on the nominated submissions. And winners are determined by voting.*  
```
The winner should be decided in the voting stage rather than the result stage. The logic here is obviously wrong.

Reviewer 4: Believe the way Voting/Results was already arranged makes more sense. The current additional clause is heavily implied by the fact a voting stage is carried out.

Author: well, winners are determined by voting is kinda a common sense, i don't think anything need to be added

> - **Results:** *Winners are determined by voting, with a livestream revealing the podium finishers in each category.*

Author: same as before

> - **Results:** *Winners are determined by voting, with a livestream revealing the podium finishers in each category.*  

Reviewer 5: ```suggestion
- **Results:** *Winners are determined by voting, with a livestream revealing the podium finishers in each category.*
```

If you’re eligible, head over to the [**Mapper’s Choice Awards 2024**](https://mca.corsace.io/2024/) website to get started!

> ---  

Reviewer 5: ```suggestion
---
```

### **Schedule**  

| Event | Date (UTC) |

> | --: | :-- |  

Author: Will try use extension to check and fix on all of them, not sure why all these whitespaces appear.

| **Nomination phase** | 2025-03-26 – 2025-03-31 00:00 |
| **Voting phase** | 2025-04-03 – 2025-04-08 00:00 |
| **Results stream** [Corsace Twitch channel](https://www.twitch.tv/corsace) | 2025-04-14 |
| **Final results** [Corsace website](https://mca.corsace.io/2024/) | 2025-04-15 |

---

### **Rewards**  

MCA 2024 features exciting rewards for the winners of each category:  

- **osu!supporter**  
  - Grand Award winners: **4 months**  
- **Badges**  
  - Special badges for **Grand Award** category winners.  
  - Generic badges for **regular category** winners. 

---

### **A Year in Mapping (AYIM) Update**  

> Unfortunately, **A Year in Mapping (AYIM)** will not be hosted this year. *As the main organisers have stepped away from the project, it will no longer be supported.* We want to extend our **heartfelt gratitude** to those who contributed over the years, making AYIM a cherished part of the community.  

Reviewer 6: ```suggestion
As we look back on 2024, the countless memories left behind in the community remain fresh in our minds. Now, an even deeper dive into the data and stats behind 2024’s mapping trends awaits you in **A Year In Mapping (AYIM) 2024**！
```

Reviewer 4: ```suggestion
As we look back on 2024, the countless memories left behind in the community remain fresh in our minds. Now, an even deeper dive into the data and stats behind 2024's mapping trends awaits you in **A Year In Mapping (AYIM) 2024**!
```

> 

Reviewer 6: ```suggestion
As always, **A Year in Mapping** serves as the community’s annual report, presenting [the most comprehensive and exciting stats of 2024](https://lookerstudio.google.com/u/0/reporting/aa599b20-f74b-4699-9e69-504f542de5aa/page/hj1uB?s=vvKt9IOmetM). Want to know how many ranked maps and mappers each mode had? Or which songs and artists made the biggest impact on the community this year? It’s all inside AYIM — take a look and discover!
```

Reviewer 4: ```suggestion
As always, **A Year in Mapping** serves as the mapping community's annual report, presenting [the most comprehensive and exciting stats of 2024](https://lookerstudio.google.com/u/0/reporting/aa599b20-f74b-4699-9e69-504f542de5aa/page/hj1uB?s=vvKt9IOmetM). Want to know how many ranked maps and mappers each mode had? Or perhaps which songs and artists made the biggest impact on the community this year? It’s all inside AYIM — take a look for yourself!
```

Now that you’re caught up on all the details, it’s time for the most important part—**nominating and voting!** [**Visit the website to get started!**](https://mca.corsace.io/2024/)  

> —Maxus & momoyo  

Reviewer 6: ```suggestion
—Maxus, momoyo, Seiran-
```

Reviewer 7: credits are for original authors, not for reviewers

Reviewer 6: > credits are for original authors, not for reviewers

I have negotiated with maxus. First of all, I am responsible for the content of AYIM. Second, I helped to correct a lot of content, not just the format. It was just because I used GitHub for the first time and was not familiar with the process, so I used a reviewer. If this cannot be counted as participation credit, I think it is not very reasonable.

Reviewer 6: I think some invisible work that is not shown on GitHub needs to be communicated

Reviewer 4: Please communicate things through the osu-news thread... we made it precisely to communicate things through there. This does not need to be handled through DMs.

Author: Next time i would appreciate if you let me and momoyo to handle this first before immediately adding your name in here as suggestion. This overcomplicate and causes potential misunderstanding. 

I need to contact momoyo first and then summarize everything that gonna be changed as to make things simpler, so next time please don't do this kind of suggestion, we will make manual changes on our own for this depending on what you contribute.
